Makefile

```diff
@@ -33,6 +33,9 @@ fix-cs:
 stan:
 	docker-compose run --rm --no-deps app php -d memory_limit=1G ./vendor/bin/phpstan analyse --level=9 --no-progress src/ tests/
 
-setup: install-php
+generate-test-inspectors:
+	docker-compose run --rm --no-deps app php -f ./tests/cli/bootstrap.php && ./vendor/bin/phpcbf ./tests/Fixtures
```

Comment:
Could you check with the updated inspector generator if this is still necessary? If the generated class are incompatible with our CS, then I'd rather generate them into a different directory in `tests` and exclude that dir from php-cs